### ISSUE TITLE
Migration: create_reviews - colon move

### ISSUE THREAD

**Issue by onyoo** [learn-co-curriculum/flatiron-bnb-methods#8]
/db/migrate/20140709193156_create_reviews.rb: 8 , reads:

```t.timestamps :null false```

when a rake is attempted, this error is thrown:

```
rake aborted!
SyntaxError: /Users/Roberto/Development/code/flatiron-bnb-methods-v-000/db/migrate/20140709193156_create_reviews.rb:8: syntax error, unexpected keyword_false, expecting keyword_end
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `block in require'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:774:in `load_migration'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:770:in `migration'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:765:in `disable_ddl_transaction'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:1048:in `use_transaction?'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:1040:in `ddl_transaction'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:994:in `execute_migration_in_transaction'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:956:in `block in migrate'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:952:in `each'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:952:in `migrate'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:820:in `up'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/migration.rb:798:in `migrate'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/tasks/database_tasks.rb:137:in `migrate'
/Users/Roberto/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.0/lib/active_record/railties/databases.rake:44:in `block (2 levels) in <top (required)>'
/Users/Roberto/.rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `eval'
/Users/Roberto/.rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

Fix to get migrations running:

```t.timestamps null: false```

**Comment by NoneOfMaster** [comment]
seconded. just recreated the problem and the fix.

**Comment by AnnJohn** [comment]
Thank you @onyoo and @NoneOfMaster. We'll fix this within the next few weeks.

**Comment by pletcher** [comment]
Fixed in #11
